.github/workflows/scheduled_cleanup.yaml

```diff
@@ -0,0 +1,35 @@
+name: Scheduled Cleanup repo
+
+on:
+  push:
+  schedule:
+    - cron: '30 00 * * sun'
+  workflow_dispatch:
+    inputs:
+        number_days_old:
+            description: "pass this value, all the files older than this value will be deleted from our repository. By default, this value is set to 30 days."
+            required: false
+            default: 30
+        artifactory_name:
+            description: "deletion by the name of the artifactory_name"
+            required: false
+
+jobs:
+  jfrog_packages_cleanup:
+     name: install jfrog cli
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - uses: jfrog/setup-jfrog-cli@v2
+         env:
+           JF_ENV_DEVOPS: ${{secrets.JF_SECRET_ENV_DEVOPS}}
+       - run: |
+           if [ -n ${{inputs.artifactory_name}} ]; then
```

Comment:
if the user has enter an artifactory name then the line `"name": {"$match":"*"},` will be replaced by:
`"name": {"$match":"${{inputs.artifactory_name}}"},

this is possible thanks to:
`"name":.*` This will replace "name": and all of the rest of the line.

Reply:
I don't think this is good enough for our case. We want to delete all the artifacts that have been built from a branch given in input. With this match I need to specify exactly the name and version